pygac/reader.py

```diff
@@ -734,8 +736,10 @@ def get_sat_angles_without_tle(self):
             sat_alt,  # approximate satellite altitude
             self.times[:, np.newaxis],
             self.lons, self.lats, 0)
-        # Sometimes the get_observer_look_not_tle returns nodata instead of 90.
-        sat_elev[:, mid_column] = 90
+        # Sometimes (pyorbital <= 1.6.1) the get_observer_look_not_tle returns nodata instead of 90.
+        # Problem solved with https://github.com/pytroll/pyorbital/pull/77
+        if LooseVersion(pyorbital.__version__) <= LooseVersion('1.6.1'):
+            sat_elev[:, mid_column] = 90
```

Comment:
Could you issue a warning here saying eg that for best results, pyorbital > 1.6.1 is needed?

Reply:
Results are not better with pyorbital > 1.6.1. With the fix they are the same (= 90). Without the fix they sometimes get rounded to 89.999999 instead of 90 but I don't think that is better. I added a test to check sat_elev at nadir are 90 degrees.